src/bridge/mockHelpers.js

```diff
@@ -150,6 +161,12 @@ export const scanAccounts: $PropertyType<CurrencyBridge, "scanAccounts"> = ({
         account.name = validateNameEdition(account);
         if (isLast) {
           account.spendableBalance = account.balance = BigNumber(0);
+
+          const perFamilyOperation = perFamilyMock[currency.id];
+          const clearAccount =
+            perFamilyOperation && perFamilyOperation.clearAccount;
+
+          if (clearAccount) clearAccount(account);
```

Comment:
In term of semantic I don't really understand why it's called clearAccount.

I think maybe we could have a general "enhancer" for all accounts that get scanned. or even more generally a way for you to overide what is getting generated.

to start with it could just be a `postAccountScanned: (Account, { isEmpty }) => ?Account` which you only apply for the case of an empty?